action-templates/actions/action-filter/action.yml

```diff
@@ -0,0 +1,38 @@
+name: "Path Filter"
+description: "Paths filter with input of filters"
+inputs:
+  filters:
```

Comment:
das macht keinen sinn überhaupt als input parameter zu machen. weil ich weiß leider nicht, wie man den output dynamisch macht. also der Output ist ja jetzt schon java, javascript und python

Reply:
Könnte das vielleicht mit Matrix gehen: https://docs.github.com/de/actions/writing-workflows/choosing-what-your-workflow-does/passing-information-between-jobs#verwenden-von-auftragsausgaben-in-einem-matrixauftrag

Reply:
aber da hat man schon in der yml `jobs.job1.outputs.output_*` definiert. output_1, output_2, output_3